acceptance/tests/windows/eventlog.rb

```diff
@@ -12,16 +12,16 @@
   # get remote time
   now = on(agent, "#{ruby_command(agent)} -e \"puts Time.now.utc.strftime('%m/%d/%Y %H:%M:%S')\"").stdout.chomp
 
-  # it should fail to start since parent directories don't exist
-  confdir = "/does/not/exist"
-
   # generate an error, no master on windows boxes
-  on agent, puppet_agent('--server', '127.0.0.1', '--test', '--confdir', confdir), :acceptable_exit_codes => [1]
+  # we use `agent` because it creates an eventlog log destination by default,
+  # whereas `apply` does not.
+  on agent, puppet('agent', '--server', '127.0.0.1', '--test'), :acceptable_exit_codes => [1]
 
   # make sure there's a Puppet error message in the log
   # cygwin + ssh + wmic hangs trying to read stdin, so echo '' |
   on agent, "cmd /c echo '' | wmic ntevent where \"LogFile='Application' and SourceName='Puppet' and TimeWritten >= '#{now}'\"  get Message,Type /format:csv" do
-    fail_test "Event not found in Application event log" unless
-      stdout =~ /Cannot create [a-z]:\/does\/not\/exist.*,Error/mi
+    if stdout !~ /Could not retrieve catalog.*skipping run,Error/mi
+      fail_test "Event not found in Application event log"
+    end
   end
```

Comment:
I prefer `if something fail_test`, but there are lots of tests that use `fail_test unless`. I'll put that part back.

Historically beaker has supported "global" `stdout`, `stderr` methods that are mixed into the test case. While I too dislike that (and prefer for the `result` to be yielded to the block`), there are many tests that still follow the old convention, and I would rather fix all of them at once at a different time:

```
$ git grep ' stdout' tests
tests/aix/aix_package_provider.rb:    actual_version = stdout.chomp
tests/aix/aix_package_provider.rb:  assert_match(/aix package provider is unable to downgrade packages/, stdout, "Didn't get an error about downgrading packages")
tests/aix/nim_package_provider.rb:    actual_version = stdout.chomp
tests/aix/nim_package_provider.rb:        assert_match(/NIM package provider is unable to downgrade packages/, stdout, "Didn't get an error about downgrading packages")
tests/allow_arbitrary_node_name_fact_for_agent.rb:  node_names << stdout.chomp
tests/allow_arbitrary_node_name_fact_for_agent.rb:    assert_match(/defined 'message'.*#{success_message}/, stdout)
tests/allow_arbitrary_node_name_fact_for_apply.rb:  node_names << stdout.chomp
tests/allow_arbitrary_node_name_fact_for_apply.rb:  assert_match(/defined 'message'.*#{success_message}/, stdout)
tests/allow_arbitrary_node_name_for_agent.rb:    assert_match(/defined 'message'.*#{success_message}/, stdout)
tests/allow_arbitrary_node_name_for_apply.rb:  assert_match(/defined 'message'.*#{success_message}/, stdout)
tests/allow_symlinks_as_config_directories.rb:    assert_match(/My certname is awesome_certname[^\w]/, stdout)
tests/apply/classes/parameterized_classes.rb:    fail_test "inclusion after parameterization failed" unless stdout.include? "1-2"
...
```